app/javascript/components/widgets/weather/daily-weather.jsx

```diff
@@ -14,7 +14,7 @@ const Wrapper = styled(Row)`
   color: ${colors.grey};
   font-weight: ${weights.light};
   margin-left: ${spacing.xl};
-  margin-top: ${spacing.xxl};
+  margin-top: 65px;
```

Comment:
Is this a one off or should it become the new `spacing.xxl` value?

Reply:
I'm not in favor of modifying spacing constants unless I get buy-in from a designer and carefully check all instances in the app/site that would be affected. (Also, if it were simply added, it'd be need to be greater than the current largest `xxxl: '40px',` -- `xxxxl: '65px'` feels unweildy to me).

Reply:
I was thinking that maybe sometime we could talk with the designers about that, because a lot of the spacing given in the design does not lend itself to reusability well.

Reply:
Agreed. I meant this as more of a question to discover design intent versus asking for a change in the PR.